### DISCUSSION TITLE
是否支持调用本地 ollama服务？？

### DISCUSSION THREAD

**Discussion by geminizyz** [dataelement/bisheng#517] [Q&A]
是否支持调用本地 ollama服务(没有装rt)？？ 应该怎么配置技能？ 我用customllm +  conversationchain，前台对话一直转圈圈，docker logs bisheng-backend 报错如下 
![微信截图_20240429191958](https://github.com/dataelement/bisheng/assets/167763677/975dd3c6-57f6-4ab5-a599-b2029ceb31ad)
![微信截图_20240429192638](https://github.com/dataelement/bisheng/assets/167763677/8abefda5-0b3a-483f-9ddb-8fb1a270dc1d)
ollama本身没有问题（fastgpt)

**Comment by FILAgiao** [comment]
同问，我如果现在想使用ollama服务，我需要重写llmchain，因为ollama会分段返回，只有设置stream=false才可以，加个变量导致要重写llmchain，我要怎么做？

**Comment by wisdomqin** [comment]
现在已经支持所有主流开源推理框架了~
